src/Newsletter/ProviderHandler/Newsletter2Go.php

```diff
@@ -0,0 +1,373 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: tmittendorfer
+ * Date: 10.07.2018
+ * Time: 13:16
+ */
+
+namespace CustomerManagementFrameworkBundle\Newsletter\ProviderHandler;
+
+
+use CustomerManagementFrameworkBundle\Model\Activity\Newsletter2GoStatusChangeActivity;
+use CustomerManagementFrameworkBundle\Model\NewsletterAwareCustomerInterface;
+use CustomerManagementFrameworkBundle\Newsletter\ProviderHandler\Newsletter2Go\Newsletter2GoExportService;
+use CustomerManagementFrameworkBundle\Newsletter\Queue\Item\DefaultNewsletterQueueItem;
+use CustomerManagementFrameworkBundle\Newsletter\Queue\Item\NewsletterQueueItemInterface;
+use CustomerManagementFrameworkBundle\Newsletter\Queue\NewsletterQueueInterface;
+use CustomerManagementFrameworkBundle\Traits\LoggerAware;
+
+class Newsletter2Go implements NewsletterProviderHandlerInterface
+{
+    use LoggerAware;
+
+
+    /**
+     * @var string
+     */
+    protected $shortcut;
+
+    /**
+     * @var string
+     */
+    protected $listId;
+
+    /**
+     * @var Newsletter2GoExportService
+     */
+    protected $exportService;
+
+
+    /**
+     * @var array
+     */
+    protected $mergeFieldMapping;
+
+    /**
+     * @var array
+     */
+    protected $fieldTransformers;
+
+    /**
+     * @var array $statusMapping
+     * @var array $reverseStatusMapping
+     */
+    protected $statusMapping;
+    protected $reverseStatusMapping;
+
+    protected $doubleOptInFormCode;
+
+
+    public function __construct($shortcut, $listId, $doubleOptInFormCode, array $statusMapping = [], array $reverseStatusMapping = [], array $mergeFieldMapping = [], array $fieldTransformers = [], Newsletter2Go\Newsletter2GoExportService $exportService)
```

Comment:
param with no default value after params with default values ... does not make any sense